workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/utilities/TokenUtilities.kt

```diff
@@ -4,17 +4,16 @@ package com.r3.corda.lib.tokens.workflows.utilities
 
 import com.r3.corda.lib.tokens.contracts.states.NonFungibleToken
 import com.r3.corda.lib.tokens.contracts.types.IssuedTokenType
-import com.r3.corda.lib.tokens.contracts.types.TokenType
 import net.corda.core.contracts.UniqueIdentifier
 import net.corda.core.identity.AbstractParty
 
 /**
  * Creates a [NonFungibleToken] from an [IssuedTokenType].
- * E.g. IssuedTokenType<TokenType> -> NonFungibleToken<TokenType>.
+ * E.g. IssuedTokenType<TokenType> -> NonFungibleToken.
  * This function must exist outside of the contracts module as creating a unique identifier is non deterministic.
  */
-infix fun <T : TokenType> IssuedTokenType<T>.heldBy(owner: AbstractParty): NonFungibleToken<T> = _heldBy(owner)
+infix fun IssuedTokenType.heldBy(owner: AbstractParty): NonFungibleToken = _heldBy(owner)
```

Comment:
Why do i see this function for a third time :joy:

Reply:
Has to be here as we can't use `UniqueIdentifier` inside the `contracts` module.